perception/object_filter/launch/detected_object_filter.launch.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="vector_map_topic" default="/map/vector_map"/>
+  <arg name="input/object" default="in_objects"/>
+  <arg name="output/object" default="out_objects"/>
+
+  <node pkg="object_filter" exec="detected_object_filter" name="detected_object_filter" output="screen">
+    <remap from="input/vector_map" to="$(var vector_map_topic)"/>
+    <remap from="input/object" to="$(var input/object)"/>
+    <remap from="output/object" to="$(var output/object)"/>
+    <param name="filter_by_xy_position" value="false"/>
+    <param name="upper_bound_x" value="100.0"/>
+    <param name="lower_bound_x" value="0.0"/>
+    <param name="upper_bound_y" value="10.0"/>
+    <param name="lower_bound_y" value="-10.0"/>
```

Comment:
I think this parameter needs to be adjusted for each product, so could you parameterize these value?

Reply:
Add parameter yaml https://github.com/autowarefoundation/autoware.universe/pull/1221/commits/71d943a7e0c9d6884bb0a38a7c6abb55e704dd96